DOC: Correct installation instructions (#3205)
uxlfoundation/oneDAL: docs/source/installation.rst

.. See the License for the specific language governing permissions and
.. limitations under the License.

Installation
============

> There are several options available for installing oneDAL:

Reviewer: ```suggestion
There are several options available for installing |short_name|:
```

- **Binary Distribution**: pre-built binary packages are available from the following sources:

  - Intel® oneAPI:

>     - Download as Part of the `Intel® oneAPI Base Toolkit <https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html>`__

Reviewer: I'm not sure that it works as part of the reference, but you can try:
```suggestion
    - Download as Part of the `|base_tk| <https://www.intel.com/content/www/us/en/developer/tools/oneapi/base-toolkit-download.html>`__
```

Author: Looks like it didn't work, will investigate:
![image](https://github.com/user-attachments/assets/bbb0b615-ad60-4c79-ac16-10dd64bcf467)

Author: Fixed now.

    - Download as the Stand-Alone `oneAPI Data Analytics Library <https://www.intel.com/content/www/us/en/developer/tools/oneapi/onedal-download.html>`__
  - Conda: ::

      conda install -c conda-forge dal-devel

  - `NuGet <https://www.nuget.org/packages/inteldal.devel.linux-x64>`__

> - **Source Distribution**: Clone the `GitHub repository <https://github.com/uxlfoundation/oneDAL>`__ or `download a specific version of oneDAL <https://github.com/uxlfoundation/oneDAL/releases>`__ from the GitHub releases page and follow the instructions in the `INSTALL.md file <https://github.com/uxlfoundation/oneDAL/blob/main/INSTALL.md>`__.

Reviewer: ```suggestion
- **Source Distribution**: Clone the `GitHub repository <https://github.com/uxlfoundation/oneDAL>`__ or `download a specific version of |short_name| <https://github.com/uxlfoundation/oneDAL/releases>`__ from the GitHub releases page and follow the instructions in the `INSTALL.md file <https://github.com/uxlfoundation/oneDAL/blob/main/INSTALL.md>`__.
```
